code/datums/ammo/bullet/pistol.dm

```diff
@@ -180,12 +180,10 @@
 	headshot_state = HEADSHOT_OVERLAY_MEDIUM
 	debilitate = list(0,0,0,0,0,0,0,2)
 
-	effective_range_max = 3
```

Comment:
Just increase this value then. As is this makes it unlimited effective range.

Reply:
Removing this brings the pistol in-line with every other Pistol in the game barring one civilian version. 

What value would you suggest if one were to be instituted?

Reply:
~6?